File: ACOIIN 2025 /Semana 1/Python/d.py
total = 0

for i in range(0, 5) :
    a = int(input())
    b = int(input())
    c = int(input())
    
    ans = a + b + c
    total += ans
    
    print(f"Dia {i + 1}: ${ans}")
    
print(f"Total semanal: ${total}")
print(f"Promedio por dia: ${total / 5}")

File: ACOIIN 2025 /Semana 1/C++/d.cpp
#pragma GCC optimize("O2")
#include <bits/stdc++.h>
using namespace std;
#define ll long long
#define f float
#define endl '\n'

void usaco(string filename) {
    freopen((filename + ".in").c_str(), "r", stdin);
    freopen((filename + ".out").c_str(), "w", stdout);
}

int main() {
    ios::sync_with_stdio(0);
    cin.tie(0);
    cout.setf(ios::fixed);
    cout.precision(10);

    // usaco("filename");

    ll total = 0;
    for(int i = 1; i <= 5; i++) {
        int a, b, c;
        cin >> a >> b >> c;

        int temp = a + b + c;
        total += temp;

        cout << "Dia " << i << ": $" << temp << endl;
    }

    cout << "Total: $" << total << endl;
    cout << "Promedio: $" << total / 5 << endl;

    return 0;
}